tools/apidoc/generatecommands.xsl

```diff
@@ -57,7 +57,7 @@ version="1.0">
 				<xsl:for-each select="command/command">
 					<!-- Modify this line for the release version -->
 					<span>
-									Apache CloudStack v4.9.0 Root Admin API Reference
+									Apache CloudStack v%ACS_RELEASE% Root Admin API Reference
```

Comment:
do we need/want the `v`? It is always `v` anyways.

Reply:
Good point. I agree with you @rafaelweingartner. Updated it!